template/php/template.yml

```diff
@@ -0,0 +1,2 @@
+language: php
```

Comment:
Do we want `php` or `php7`?

Reply:
So, the image will auto-roll up the minor versions of 7 - it's a question of, what do we do when there's php8?

We'll likely not support php5 as there's just no good reasoning for writing new code under that. Personally, I'd like to see us go php7 and version the templates - maybe there's a bigger architectural view there though? the language then sub-directories of supported versions? Not sure.

Reply:
Agree with being explicit in the template name, `php7`, doesn't commit us to supporting anything now, but makes the introduction of future templates easier.

As for exposing versions via the tool (`--lang php --lang-version 7`) I think thats decoupled from where they get stored in the template repo, you'd ultimately want to have a per-version dir for the template and would just be abstracting away the choice.

Reply:
Are we all in agreement this is to be renamed php7 and to always be the latest stable php release?

Reply:
+1 to php7 rename

Reply:
+1 to php7 rename

Reply:
Aye, php7 is the way to go